cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+function cleanup() {
+  echo "[customize-image] cleanup"
+  rm -rf "$CLOUD_INIT_ISO"
+  virsh destroy "$DOMAIN_NAME" || true
+  virsh undefine "$DOMAIN_NAME" || true
+
+  if [ $? -ne 0 ]; then
+    rm -f "$NEW_IMAGE"
+  fi
+}
+
+IMAGE_PATH=$1
+NEW_IMAGE=$2
+CLOUD_CONFIG_PATH=$3
+
+export OS_VARIANT=${OS_VARIANT:-fedora31}
+
+readonly DOMAIN_NAME="provision-vm"
+readonly CLOUD_INIT_ISO="cloudinit.iso"
+
+if ! [ -f "${CLOUD_CONFIG_PATH}" ]; then
+  CLOUD_CONFIG_PATH="user-cloud-config"
```

Comment:
I do not understand this. You use the same variable to provide the file instead of the provided path?

Reply:
If the file path of `CLOUD_CONFIG_PATH` does not exist its sets `CLOUD_CONFIG_PATH` default file path.
There is an error here because the file "user-cloud-config" is not there I forgot to add it,
I should set `CLOUD_CONFIG_PATH` default to `example/example-cloud-config` file